### PR TITLE
[11.0][MIG] purchase_order_type: Migration to 11.0

### PR DESCRIPTION
**Pull request by oihane**

### CODE REVIEW

File: purchase_order_type/__init__.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline purchase_order_type/__init__.py]: @oihane no need to here

File: purchase_order_type/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline purchase_order_type/__manifest__.py]: @oihane Copyright should be 

```
# Copyright (C) 2015 Camptocamp SA
# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

File: purchase_order_type/models/purchase_order.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```
Comment (nikul-serpentcs) [inline purchase_order_type/models/purchase_order.py]: same here or Other files

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Improve Code

File: purchase_order_type/__init__.py (hunk 1 above)
Comment (oihane) [reply purchase_order_type/__init__.py]: Done

File: purchase_order_type/__manifest__.py (hunk 2 above)
Comment (oihane) [reply purchase_order_type/__manifest__.py]: I didn't have any idea if I could change it.

File: purchase_order_type/models/purchase_order.py (hunk 3 above)
Comment (oihane) [reply purchase_order_type/models/purchase_order.py]: done, in every py file

**Review by anajuaristi — APPROVED** [review]

**Review by anajuaristi — APPROVED** [review]

**Comment by tbaden** [comment]
@nikul-serpentcs your changes seem resolved. anything else or good to go?
@oihane please squash your commits into one migration commit

**Review by nikul-serpentcs — APPROVED** [review]
Only code Review LGTM :+1:

**Comment by HviorForgeFlow** [comment]
Code Review
Why is defined a relation between Partner an Purchase Type? What is the use case?

**Comment by oihane** [comment]
@hveficent is similar to `sale_order_type`, you can define a default purchase type in a supplier if you select that supplier the purchase order will have the type defined in the supplier instead of the default one.

**Review by HviorForgeFlow — APPROVED** [review]

**Pull request merged**